src/mbgl/map/source.cpp

```diff
@@ -39,9 +39,7 @@ Source::Source() {}
 Source::~Source() = default;
 
 bool Source::isLoaded() const {
-    if (!loaded) {
```

Comment:
Agreed about `enabled`, but I believe we also should check if there is a pending request, which indicates the source is in loading state.